src/main/java/com/google/cloud/tools/gradle/appengine/core/AppEngineCorePluginConfiguration.java

```diff
@@ -93,6 +93,10 @@ private void createDownloadCloudSdkTask() {
                   p -> {
                     downloadCloudSdkTask.setCloudSdkBuilderFactory(cloudSdkBuilderFactory);
                     downloadCloudSdkTask.setManagedCloudSdkFactory(managedCloudSdkFactory);
+                    downloadCloudSdkTask.setProgressListener(
```

Comment:
Could `DownloadCloudSdkTask` make its own `ProgressListener` and `ConsoleListener`?

Reply:
Hmm, yeah I guess they could. I originally set them externally in case they needed to be mocked in a test, but I can't see much reason to test them the way they are right now. If we end up needing to customize the progress output more, then it may end up being better this way for testing. For now I can simplify it.